app/src/main/java/io/intrepid/contest/screens/contestcreation/ContestActivityContract.java

```diff
@@ -0,0 +1,14 @@
+package io.intrepid.contest.screens.contestcreation;
+
+import java.util.List;
+
+import io.intrepid.contest.models.Category;
+
+public interface ContestActivityContract {
```

Comment:
Lets rename this to NewContestContract, I think its a bit more clear and decouples it from the term Activity